app/src/main/java/com/zulip/android/models/Message.java

```diff
@@ -115,6 +116,8 @@
     private Stream stream;
     @DatabaseField(columnName = MESSAGE_READ_FIELD)
     private Boolean messageRead;
+    @DatabaseField(columnName = MESSAGE_STAR_FIELD)
+    private Boolean messageStar;
```

Comment:
@Sam1301 `messageStar` is already of type boolean. I don't get it exactly. Can you please correct me?